doctrine/dbal.rst

```diff
@@ -13,8 +13,8 @@ How to Use Doctrine DBAL
 
 The `Doctrine`_ Database Abstraction Layer (DBAL) is an abstraction layer that
 sits on top of `PDO`_ and offers an intuitive and flexible API for communicating
-with the most popular relational databases. In other words, the DBAL library
-makes it easy to execute queries and perform other database actions.
+with the most popular relational databases. In other words, executing queries and
+performing other database actions can be achieved with the DBAL library.
```

Comment:
i would change the last sentence to "The DBAL library allows you to write queries independently of your ORM models, e.g. for building reports or direct data manipulations."